src/edwards.rs

```diff
@@ -269,10 +269,76 @@ impl ExtendedPoint {
         }
     }
 
-    /// Convert this point to its equivalent on the Montgomery form of the
-    /// curve.
+    /// Convert this `ExtendedPoint` on the Edwards model to the
+    /// corresponding `MontgomeryPoint` on the Montgomery model.
+    ///
+    /// Note that this is a one-way conversion, since the Montgomery
+    /// model does not retain sign information.
+    ///
+    // XXX need to figure out how to keep this in internal docs, and
```

Comment:
why does this need to be in internal docs?

Reply:
my reasoning was that the formulas that it uses internally are an implementation detail, not something the API consumer has to know about.